### PR TITLE
remove node version from installation instructions

### PR DESCRIPTION
**Pull request by pwamej**
It's outdated (v16 is the active LTS curently) and IMO it's better to link to the Node docs rather than update it every 6 months.

#### :heavy_check_mark: Checklist

- [X] Added or updated documentation
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ guide to do a repository-based installation.
   [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)
 - An account with elevated rights to install the dependencies
 - `curl` or `wget` installed
-- Node.js Active LTS Release installed (currently v14) using one of these
+- [Node.js Active LTS Release](https://nodejs.org/en/about/releases/) installed using one of these
```
Comment (bassco) [inline docs/getting-started/index.md:28]:
```suggestion
Node.js [Active LTS Release](https://nodejs.org/en/about/releases/) installed using one of these
```

To keep the format you are introducing consistent.

Comment (pwamej) [reply docs/getting-started/index.md]: right, thanks!

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ guide to do a repository-based installation.
   [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)
 - An account with elevated rights to install the dependencies
 - `curl` or `wget` installed
-- Node.js Active LTS Release installed (currently v14) using one of these
+Node.js [Active LTS Release](https://nodejs.org/en/about/releases/) installed using one of these
```
Comment (Rugvip) [inline docs/getting-started/index.md:28]: Should still be a bullet here right?

**Review by Rugvip — COMMENTED** [review]
Thank you, makes sense!

Just needs a tiny fix

File: docs/getting-started/index.md (hunk 2 above)
Comment (pwamej) [reply docs/getting-started/index.md:28]:
```suggestion
- Node.js [Active LTS Release](https://nodejs.org/en/about/releases/) installed using one of these
```
Oh right, thanks

**Review by Rugvip — APPROVED** [review]
Thanks!

Needs DCO then we can :shipit:

**Pull request merged**